openpype/hosts/blender/api/ops.py

```diff
@@ -1016,27 +1020,26 @@ def __init__(self):
         if not bpy.app.timers.is_registered(_process_app_events):
             bpy.app.timers.register(_process_app_events, persistent=True)
 
-    def _build_first_workfile(self):
-        # clear all objects and collections
-        for obj in set(bpy.data.objects):
-            bpy.data.objects.remove(obj)
-        for collection in set(bpy.data.collections):
-            bpy.data.collections.remove(collection)
-        # purgne unused datablock
-        while bpy.data.orphans_purge(do_local_ids=False, do_recursive=True):
-            pass
-        # clear all libraries
-        for library in list(bpy.data.libraries):
-            bpy.data.libraries.remove(library)
+    def _build_first_workfile(self, clear_scene, save_as):
```

Comment:
Could you document this please? Hints and docstring

Reply:
yep

Reply:
you forgot hints :)

Reply:
oups

Reply:
done :)